### PR TITLE
Reorganize project to support editable installation

### PR DESCRIPTION
**Pull request by aaron-kaplan**
I've made no functional changes yet, just reorganizing the directory structure to support [editable installation](https://setuptools.pypa.io/en/latest/userguide/development_mode.html), which I find makes development a lot easier. With an editable installation, you can edit a python file, restart your notebook kernel, and immediately see the change, without rebuilding and reinstalling the package first.

Also removed one copy of a function that was defined twice, and corrected some errors in the README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,39 +2,52 @@
...
-git clone https://github.com/kjhall-iri/cpt-tools.git
-cd cpt-tools
+git clone https://github.com/iri-pycpt/cpt-io.git
+cd cpt-io
 pytest -v .
 ```
 
 
-for support: kjhall@iri.columbia.edu 
+for support: awr@iri.columbia.edu
```
Comment (aaron-kaplan) [inline README.md:52]: @awrobertson is there a group email we can use so that this doesn't have to be updated each time we have staff changes?

Comment (awrobertson) [reply README.md:52]: Not that I recall. Please check with Jeff to create one.

**Comment by awrobertson** [comment]
This looks great to me Aaron. Please go ahead and merge.
In future Jeff and Bohar could be added as reviewers, increasing the chances of an informed review.

File: README.md (hunk 1 above)
Comment (aaron-kaplan) [reply README.md:52]: What should we call it? pycpt@iri.columbia.edu? pycpt-help@iri.columbia.edu? Who should receive the emails?

Comment (aaron-kaplan) [reply README.md:52]: I'm going to merge this now so as not to hold up further changes, but I'll be sure to update the email address before releasing a new vesrion.

**Pull request merged**